subsys/mgmt/mcumgr/smp_bt.c

```diff
@@ -53,6 +53,9 @@ static ssize_t smp_bt_chr_write(struct bt_conn *conn,
 	struct net_buf *nb;
 
 	nb = mcumgr_buf_alloc();
+	if (!nb) {
+		return -1;
+	}
```

Comment:
Could some error code be used here?

Reply:
I left this intentionally to start this discussion. I totally agree `-1` is not what is good here but i saw similar thing here
https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/mgmt/mcumgr/smp_bt.c#L169. I could change this in both places but wanted to discuss it earlier.
What do you think about including `<mgmt/mgmt.h>` and returning `-MGMT_ERR_ENOMEM` here and `-MGMT_ERR_EINVAL` at line 169.

Reply:
OK, `-MGMT_ERR_ENOMEM` should be fine